flow/envs/base_env.py

```diff
@@ -131,6 +134,78 @@ def __init__(self, env_params, sumo_params, scenario):
         self.start_sumo()
         self.setup_initial_state()
 
+        mode = self.sumo_params.render
```

Comment:
This segment could use some comments, it's a bit hard to read right now

Reply:
Solved.